src/components/dashboard/ClaimButton.js

```diff
@@ -146,22 +146,22 @@ const ClaimAnimationButton = memo(({ styles, entitlement, nextClaim, onPress, is
     }
   }, [entitlement])
 
-  if (initialEntitlementRef.current) {
-    return (
-      <ClaimButton
-        {...buttonProps}
-        styles={styles}
-        entitlement={entitlement}
-        nextClaim={nextClaim}
-        onPress={onPress}
-        isInQueue={isInQueue}
-      />
-    )
-  }
+  // if (initialEntitlementRef.current) {
+  //   return (
+  //     <ClaimButton
+  //       {...buttonProps}
+  //       styles={styles}
+  //       entitlement={entitlement}
+  //       nextClaim={nextClaim}
+  //       onPress={onPress}
+  //       isInQueue={isInQueue}
+  //     />
+  //   )
+  // }
 
-  if (!entitlement || isInQueue) {
-    return <ClaimButton styles={styles} {...buttonProps} nextClaim={nextClaim} isInQueue={isInQueue} />
-  }
+  // if (!entitlement || isInQueue) {
+  //   return <ClaimButton styles={styles} {...buttonProps} nextClaim={nextClaim} isInQueue={isInQueue} />
+  // }
```

Comment:
```suggestion
  if (!entitlement || isInQueue) {
    return <ClaimButton styles={styles} {...buttonProps} nextClaim={nextClaim} isInQueue={isInQueue} />
  }
```

why that is needed ? i'm not sure to animate button during countdown is a good idea.
from my point of view, 162-164 have to be restored. 
in that case the button will be animated always if user could claim (despite if he/she has waited for the countdown ended or was already able to claim)

Reply:
I dont  understand what problem removing these line causes?
from my point of view its just code repeating.

Reply:
> I dont understand what problem removing these line causes?
> from my point of view its just code repeating.

if remove those lines animation will be performed during countdown.
the condition checks is user ready to claim. if not ready yet it renders the non-animated button with countdown

Reply:
if you would omit code repeating we could do like that - see my latest suggestions

Reply:
i still dont understand.
the current functionality I experience when i test this
1. user  can't claim yet -> enters claim page -> countdown is shown without animation
2. user can claim -> enters claim page -> flip animation is shown
3. user in claim page -> count down reaches 0 -> flip animation is shown

i add a fix that in case 2. will show the timer button before flip in orange (was blue)